test/functional/config/example_dut_config.yml

```diff
@@ -7,6 +7,7 @@ type: "local"
 # password: "example_password"
 
 # env: "HTTPS_PROXY=myproxy.example:12345"
+# reboot_timeout: "reboot_timeout_in_seconds"
```

Comment:
"" can suggest that timeout should be a string, not an integer value
What do you think about removing "" or maybe set some default value to this variable and add some comment like:

#reboot_timeout: 1800 #timeout value in seconds

Reply:
That's a good idea, thanks.